app/controllers/albumsController.js

```diff
@@ -0,0 +1,9 @@
+const express = require('express');
+
+// eslint-disable-next-line new-cap
+const albumsController = express.Router();
```

Comment:
You don't need to create another instance of Router. Only add these endpoints in the `routes.js` file.